pkg/scheduler/nodeinfo/topology_info.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodeinfo
+
+import (
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/kubernetes/pkg/apis/core/helper"
+)
+
+// TopologyPair is a key/value pair, and now used to describe labels of a node
+type TopologyPair struct {
+	Key   string
+	Value string
+}
+
+// TopologyInfo denotes a mapping from TopologyPair to a string set
```

Comment:
The string set is a set of node names. Please state that in the comment. The current comment is not very helpful.

Reply:
Done.